Tests/Unit/LoggerTests/LoRaConsoleLoggerTests.cs

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.Tests.Unit.LoggerTests
+{
+    using Microsoft.Extensions.Logging;
+    using Microsoft.Extensions.Options;
+    using Moq;
+    using System;
```

Comment:
using System.* should appear before anything else

Reply:
see above